src/components/component/component.scss

```diff
@@ -12,14 +12,6 @@
 .docs {
     kompendium-markdown {
         display: block;
-        width: calc(50% - 2rem);
-    }
-}
-
-@media (max-width: $break-point-small) {
-    .docs {
-        kompendium-markdown {
-            width: 100%;
-        }
+        width: 100%;
```

Comment:
I found a place where the text was still unnecessarily wrapped (see screenshot below), and in figuring out why, I found this simpler fix.

<img width="1438" alt="Screenshot showing that the top-level description for `LimeSchemaOptions` is still wrapped at half the available width" src="https://user-images.githubusercontent.com/154725/180180710-5bd32b63-d6ff-4e62-b94e-c8431d6294d3.png">

Reply:
I think the reason we originally had that was to avoid these kids of UIs, in which the top block would become full-width and very hard to read:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/35954987/180195106-4ed43c15-710a-4bfa-b9d9-063ad6cd384b.png">
☝️ now with this fix

👇 before this
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/35954987/180195346-03eac8d7-7562-45a7-a13b-bdaebfb3c1a1.png">


But perhaps that can be achieved simply by adding a `max-width` like what we have done here:
```scss
main kompendium-guide {
    display: block;
    width: 100%;
    max-width: 60rem;
}
```
Which results in 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/35954987/180195716-4d26b27b-01c3-4fe6-9cd4-dcbd00a9d248.png">


So I added a fixup! :)

Reply:
Great! After pushing my fixup, I did actually think that perhaps it had something to do with limiting the maximum line length, but had the same thought as you: "that's better done with max-width"!

Great minds…! 😁